ide/views.py

```diff
@@ -93,29 +166,59 @@ def save_to_db(request):
         user = None
         if net_name == '':
             net_name = 'Net'
-        try:
-            # making model sharing public by default for now
-            # TODO: Prvilege on Sharing
-            if user_id:
-                user_id = int(user_id)
-                user = User.objects.get(id=user_id)
-
-            # create a new model on share event
-            model = Network(name=net_name, public_sharing=public_sharing, author=user)
-            model.save()
-            # create first version of model
-            model_version = NetworkVersion(network=model, network_def=net)
-            model_version.save()
-            # create initial update for nextLayerId
-            model_update = NetworkUpdates(network_version=model_version,
-                                          updated_data=json.dumps({'nextLayerId': next_layer_id}),
-                                          tag='ModelShared')
-            model_update.save()
-
-            return JsonResponse({'result': 'success', 'id': model.id})
-        except:
-            return JsonResponse({'result': 'error', 'error': str(sys.exc_info()[1])})
 
+        if Network.objects.filter(name=net_name).exists():
```

Comment:
net_nam is not the primary key, please use mode id as the key to filter

Reply:
How will I check using model id? If a person is creating a new model with the same model name then only I can run a query right?

Reply:
Basically, in this method I'm taking the net name from the POST request to filter from database because that's the only parameter that can be used to do this as this would be the only similarity amongst the already existing records and new models. So I can't really use model id to filter out results.